src/js/processor.js

```diff
@@ -0,0 +1,285 @@
+const constants = require('.\\constants.js');
+
+import info from '.\\infoBlock.js';
+
+// create html element with given attributes
+export function createHtml(selector, className, innerText, ...keysValues) {
+  const elem = document.createElement(selector);
+  className === undefined ? elem.className = '' : elem.className = className;
+  innerText === undefined ? elem.innerHTML = '' : elem.innerHTML = innerText;
+  if (keysValues.length !== 0) {
```

Comment:
1. what will be if you replace `className === undefined ?` to `className`
2. I like ternary operators too, but they are not readable in case of liner expression without assignment to variable. In case you don't need to use the result of ternary operator pls use `if ()` expression

Reply:
Done, but pls check out if that is what u wanted